src/ARMJIT_Memory.cpp

```diff
@@ -628,8 +640,12 @@ void Init()
     FastMem7Start = mmap(NULL, AddrSpaceSize, PROT_NONE, MAP_ANON | MAP_PRIVATE, -1, 0);
 
     MemoryBase = (u8*)mmap(NULL, MemoryTotalSize, PROT_NONE, MAP_ANON | MAP_PRIVATE, -1, 0);
+    #ifdef __APPLE__
+        MemoryFile = shm_open("melondsfastmem" + getpid(), O_RDWR|O_CREAT, 0600);
```

Comment:
this won't compile. You can't add an integer to a string literal. You need to use something like sprintf

Reply:
for some reason it doesn't give me any errors with this.

Reply:
ah yes, this is C++ stupidness. The string literal is from type const char*, so adding an integer to it moves the pointer forward. So say the process id is 2 the string will be "londsfastmem".

Reply:
Good to know.
Will fix later.

Reply:
Fixed.